src/view/styles/Microbit.css

```diff
@@ -14,16 +17,24 @@ svg.sim.grayscale {
 }
 .sim-button {
     pointer-events: none;
+    fill: "rgb(17, 17, 17)";
+}
+.sim-button:active {

```

Comment:
Are we going to give the metal strips at the bottom the same treatment (or are they not meant to be interacted with in the same way)?

Reply:
The metal strips currently don't have functionality. We would add the same treatment if we implement functionality on those in the future.